test/function/youtube.spec.ts

```diff
@@ -343,4 +343,25 @@ testFilterFunction(youtube, [
 		funcParameter: 'Fully Alive',
 		expectedValue: 'Fully Alive',
 	},
+
+	{
+		description: 'should remove "(Videoclip Oficial)" string',
+		funcParameter: 'Track Title (Videoclip Oficial)',
+		expectedValue: 'Track Title',
+	},
+	{
+		description: 'should remove "(Video Oficial)" string',
+		funcParameter: 'Track Title [Video Oficial]',
+		expectedValue: 'Track Title',
+	},
+	{
+		description: 'should remove "(Letra)" string',
+		funcParameter: 'Track Title [Letra]',
+		expectedValue: 'Track Title',
+	},
+	{
+		description: 'should remove "(Lyrics) + [Lyrics]" string',
+		funcParameter: 'Track Title (Lyrics) [Lyrics]',
+		expectedValue: 'Track Title',
+	},
```

Comment:
What is a purpose of these tests? These cases are already covered.

Reply:
Oh... My fault, sorry 
I was checking when o Youtube the people use the two languages